### DISCUSSION TITLE
Unpublished pages showing up in searchbar

### DISCUSSION THREAD

**Discussion by Ik1497** [requarks/wiki#5875] [Error / Bug Report]
When typing in the searchbar unpublished pages show up, this isn't only for wiki editors, but also for regular users who don't have access to these pages

**Comment by ghost** [comment]
I think this is only the case when the unpublished page is within a directory where the users have writing priviliges. If you move the page into an directory where the users only have read privilege, then it is not shown in search bar. Can you confirm this?
